### PR TITLE
Dockerhub push gh actions

### PR DESCRIPTION
**Pull request by jaredb96**

### CODE REVIEW

**Comment by tptignor** [comment]
Hi @jaredb96 . Just one question and one note. Question: where will we find the CVE report after these changes merge? Note: looks like tests failed. Benign, or do you have a PR update in mind?

**Comment by jaredb96** [comment]
> Hi @jaredb96 . Just one question and one note. Question: where will we find the CVE report after these changes merge? Note: looks like tests failed. Benign, or do you have a PR update in mind?

Good question, I believe we can access the report here on github but I'll confirm. If not, I'll update the ticket to have to stored using github actions. Re failing test: will address on another ticket. Want to keep these failing test resolutions on there own dedicated issue.

**Review by tptignor — APPROVED** [review]
LGTM but I should note it takes some extra review to determine what's failing and if it could be related to the PR. Hopefully merging hmda-platform#4878 helps with this.

I recommend we hold off on adding new test content until 1. we can quickly and easily determine test results for each of the 13 sub-projects, 2. we resolve all current test errors, either by fixing the tests or disabling them with proper notes added.

**Pull request merged**
